includes/Core/Util/DeveloperPluginInstaller.php

```diff
@@ -0,0 +1,97 @@
+<?php
+/**
+ * Class Google\Site_Kit\Core\Util\DeveloperPluginInstaller
+ *
+ * @package   Google\Site_Kit
+ * @copyright 2020 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Core\Util;
+
+/**
+ * Class responsible for providing the helper plugin via the automatic updater.
+ *
+ * @since n.e.x.t
+ */
+class DeveloperPluginInstaller {
```

Comment:
I missed this before, but this class should use our regular underscore-case for classes. Only 3P classes may use pascal-case.

Reply:
Oops, I missed that too 🤦‍♂